docs/resources/source_load_generator.md

```diff
@@ -14,12 +14,15 @@ A source describes an external system you want Materialize to read data from, an
 
 ```terraform
 resource "materialize_source_load_generator" "example_source_load_generator" {
-  name                = "source_load_generator"
-  schema_name         = "schema"
-  size                = "3xsmall"
-  load_generator_type = "COUNTER"
-  tick_interval       = "500ms"
-  scale_factor        = 0.01
+  name        = "source_load_generator"
+  schema_name = "schema"
+  size        = "3xsmall"
+
+  counter_options {
+    load_generator_type = "COUNTER"
```

Comment:
I actually screwed this up in my fix. `load_generator_type` should be outside the `counter_options`